util/jenkins/export_dead_locks/export_dead_locks.py

```diff
@@ -0,0 +1,114 @@
+import boto3
+from botocore.exceptions import ClientError
+import sys
+import backoff
+import pymysql
+import time
+import uuid
+import click
+import re
+import splunklib.client as client
```

Comment:
```suggestion
import splunklib.client as splunk_client
```

Can you rename this and all other instances to something like ```splunk_client``` so it's clear that it's Splunk and not aws?